src/rtc/rtc_s1_single_job.py

```diff
@@ -1726,11 +1825,27 @@ def run_single_job(cfg: RunConfig):
                 dem_raster, radar_grid_file_dict,
                 lookside, wavelength, orbit,
                 verbose=not flag_bursts_secondary_files_are_temporary)
+
+            radar_grid_file_dict_filenames = \
+                list(radar_grid_file_dict.values())
+
             if flag_bursts_secondary_files_are_temporary:
                 # files are temporary
-                temp_files_list += list(radar_grid_file_dict.values())
+                temp_files_list += radar_grid_file_dict_filenames
             else:
-                output_file_list += list(radar_grid_file_dict.values())
+                output_file_list += radar_grid_file_dict_filenames
+
+            # Save browse image (burst) using static layers
+            if (flag_save_browse and
+                    product_type == STATIC_LAYERS_PRODUCT_TYPE):
+                browse_image_filename = \
+                    os.path.join(output_dir_bursts, f'{burst_product_id}.png')
+                save_browse_static(radar_grid_file_dict_filenames[0],
```

Comment:
It seems the static image is generated based on `radar_grid_file_dict_filenames[0]`, and the list is actually the values of the dictionary `radar_grid_file_dict`. This can cause inconsistencies on the source layer for the static browse image because:

- The first element in the list `radar_grid_file_dict_filenames` can change depending on the `save_*` flags (e.g. `save_local_inc_angle`, `save_incidence_angle`, etc.).
- It would be probably okay for our SASes (i.e. using python >= 3.7) but the older python3 does not preserve order of the key-value pair in dict.

I think it would be probably okay for the static layer production for OPERA, unless we make changes on the `save_*` flags in runconfig. But I foresee that the source layers for the static layer can be different depending on the runconfig setting.

Maybe it would be a good idea to clarify what static layer is used in `browse_image_filename`?

Reply:
Thank you, @seongsujeong , I'm handling this here:

https://github.com/opera-adt/RTC/pull/56/commits/7504e4410fc035475308cc40d2a3bd25015f2dae